src/main/java/com/google/codeu/servlets/OpportunityGetServlet.java

```diff
@@ -42,19 +45,134 @@ public void doGet(HttpServletRequest request, HttpServletResponse response)
             response.getOutputStream().println("invalid id");
             return;
         }
-        Opportunity opportunity=datastore.getOpportunity(id);
-        if(opportunity==null){
+        Opportunity opportunity = datastore.getOpportunity(id);
+        if (opportunity == null) {
             response.getOutputStream().println("invalid id");
             return;
         }
-        response.getOutputStream().println("here");
 
-        response.getOutputStream().println(opportunity.getId());
-        response.getOutputStream().println(opportunity.getMinAge());
-        response.getOutputStream().println(opportunity.getMaxAge());
-        response.getOutputStream().println(opportunity.getTitle());
-        response.getOutputStream().println(opportunity.getDescription());
 
+        ServletOutputStream stream = response.getOutputStream();
+        stream.println("<html><head><link rel='stylesheet' href='./css/opportunity.css'></script></head> <body>");
```

Comment:
IMO whenever you have a large object in DB, it is best to keep back-end and front-end separate. Like in this servlet, a better option would be to send the opportunity in a json object, and on front-end anyone can render information according to the need.